### DISCUSSION TITLE
Do users have to manually change to the official tap?

### DISCUSSION THREAD

**Discussion by alt-romes** [Homebrew/discussions#851] [Tap maintenance and Homebrew development] [answered]
Hello,

Very recently I submitted a formula to the official homebrew-core.
Before this, I had a repo alt-romes/homebrew-pcalc, and instructed users to `brew tap alt-romes/pcalc` and then `brew install pcalc`
Since the package got accepted, and now users only need to do `brew install pcalc`, I was thinking about deprecating the custom tap.
I'm wondering if users have to manually untap, or execute a manual action, in order to start using the official formula instead.

Thank you,
~romes

**Comment by carlocab** [comment, marked as answer]
You can add `pcalc` to `tap_migrations.json` in your `alt-romes/pcalc` tap. You can look [here](https://github.com/brewsci/homebrew-science) or [here](https://github.com/neovim/homebrew-neovim/pull/222) for examples.

Users will have to untap `alt-romes/pcalc` on their own, but anyone who tries to `brew install alt-romes/pcalc/pcalc` should get the version in homebrew/core.
